management/http_recordings_test.go

```diff
@@ -8,89 +8,97 @@ import (
 	"testing"
 	"time"
 
-	"github.com/dnaeon/go-vcr/v2/cassette"
-	"github.com/dnaeon/go-vcr/v2/recorder"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/dnaeon/go-vcr.v3/cassette"
+	"gopkg.in/dnaeon/go-vcr.v3/recorder"
 
 	"github.com/auth0/go-auth0"
 )
 
-func setupHTTPRecordings(t *testing.T) {
+const (
+	recordingsDIR    = "./../test/data/recordings/"
```

Comment:
Changing the folder structure to be in sync with the one in the terraform provider as well and the one also suggest by golang standards project layout: https://github.com/golang-standards/project-layout/tree/master/test.